tests/e2e/test_regular_handler_under_asgi_mount_path.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from litestar import Litestar, asgi, get
+from litestar.testing import TestClient
+
+if TYPE_CHECKING:
+    from litestar.types.asgi_types import Receive, Scope, Send
+
+
+async def asgi_app(scope: Scope, receive: Receive, send: Send) -> None:
+    assert scope["type"] == "http"
+    await send(
+        {
+            "type": "http.response.start",
+            "status": 200,
+            "headers": [
+                (b"content-type", b"text/plain"),
+                (b"content-length", b"%d" % len(scope["raw_path"])),
+            ],
+        }
+    )
+    await send(
+        {
+            "type": "http.response.body",
+            "body": scope["raw_path"],
+        }
+    )
+
+
+asgi_handler = asgi("/", is_mount=True)(asgi_app)
```

Comment:
was this meant to be added at line 42?

Reply:
ahh yeah thanks, at one point I was considering adding multiple tests to the module.